src/redteam/commands/init.ts

```diff
@@ -29,8 +29,9 @@ import {
 import { doGenerateRedteam } from './generate';
 
 const REDTEAM_CONFIG_TEMPLATE = `# Red teaming configuration
-# Docs: https://promptfoo.dev/docs/red-team/configuration
+# yaml-language-server: $schema=https://promptfoo.dev/config-schema.json
```

Comment:
schema doesn't include targets and some other redteam specific transformations.

Reply:
Please re-check the PR! It does